index.html

```diff
@@ -3050,21 +3034,28 @@ <h2>Method Operations</h2>
       </p>
       <ul>
         <li>
-use the <a>verification methods</a> listed under <code>authentication</code> to
-decide whether an update/deactivate operation is allowed; or
+make use of the <code><a>controller</a></code> property.
+        </li>
+        <li>
+use the <a>verification methods</a> listed under <code><a>authentication</a></code> to
+decide whether an update/deactivate operation is allowed.
         </li>
         <li>
 use other constructs in the <a>DID Document</a> to decide this, for example,
-a verification method specified under <code>capabilityInvocation</code> could
-be used to verify the invocation of a capability to update the DID Document.;
-or
+a verification method specified under <code><a>capabilityInvocation</a></code> could
+be used to verify the invocation of a capability to update the DID Document.
         </li>
         <li>
 not use the <a>DID Document</a> at all to decide this, but have rules that are
 "built into" the method.
         </li>
       </ul>
 
+      <p>
+Each <a>DID method</a> MUST define how authorization is implemented, including
```

Comment:
Suggest more precise language here: 
> MUST define how authorization **of operations performed on DID Document** is implemented
Partially resolves #363

Reply:
Thanks @Fak3, could you note that proposed change in the related issue if not already? I'd like to note that this PR _only_ moves text around and does not change anything (except to consolidate given new structure). Proposes for changes to text affected by this PR should be made in related issues or new PRs once this is merged.